timeseries/src/autogluon/timeseries/models/multi_window/multi_window_model.py

```diff
@@ -117,7 +116,6 @@ def _fit(
 
         # Only the model trained on most recent data is saved & used for prediction
         self.most_recent_model = trained_models[0]
-        self.most_recent_model_path = trained_models[0].path
```

Comment:
`self.most_recent_model_path` wasn't updated even if the model was moved to a new location on disk, leading to problems